src/reducers/index.js

```diff
@@ -1,3 +1,4 @@
+import { combineReducers } from 'redux'
```

Comment:
I don't see why this revision needs to occur within this propType work. Is this related? If not, please restore to the master version.

Reply:
It was missing and causing linting errors.  It isn't needed because the filter reducer is combined with the event reducer in the event reducer file.  Someone else already fixed it on master.  Reverted.